leadership/README.md

```diff
@@ -57,3 +55,30 @@ for leader := <-leaderCh {
 
 A typical use case for this is to be able to always send requests to the current
 leader.
+
+## Fault tolerance
+
+Leadership returns an error channel for Candidates and Followers that you can use
```

Comment:
Why the need for an error channel?

Basically, you already know that when the channel is closed (as in, you exit the for loop) there **is** an error